src/main/java/com/spider/solomon/order/CustomerOrderDao.java

```diff
@@ -34,7 +36,13 @@ public CustomerOrder getOrderById (int orderId) {
 	 */
 	public List<CustomerOrder> getOrdersByCustomerId (int customerId) {
 		String sql = "SELECT * FROM customer_orders WHERE customer_id = ?";
-		return jdbcTemplate.query(sql, ORDER_MAPPER, customerId);
```

Comment:
Usually not part of code-review: In looking at why `getOrderById(..)` does not load items, I noticed that the method is not used. This may be used in the future with a new controller, but just wanted to point it out.